adapters/postgres/postgres.go

```diff
@@ -1296,6 +1296,10 @@ func GetQueryOperator(op string) (string, error) {
 		return "NOT LIKE", nil
 	case "nilike":
 		return "NOT ILIKE", nil
+	case "Lanc":
```

Comment:
being case-insensitive is best for use in the address bar (URL)
- from `Lanc` to `lanc`
- from `Rdesc` to `rdesc`

Reply:
Consider this just placeholders. In the end, we can choose better names for all the options.

Reply:
This is far from start, I'm having problems setting up the project locally to run the tests. Appreciate tips to get it working...

Reply:
I've prefixed all the operators with `ltree` (lack of a better name). 
Let me know if you have better names for them.